kili/cli/project/label.py

```diff
@@ -48,6 +40,13 @@ def type_check_label(key, value):
     help="Name of the model that generated predictions, " "if labels are sent as predictions",
 )
 @Options.verbose
+@click.option(
+    "--input-format",
+    type=str,
+    help="Format in which the labels are encoded",
+    default="raw",
```

Comment:
Isn't better to call it 'kili' ?

Reply:
there is kili "raw" and kili "simple" in the UI, so I did not change that.